packages/web-components/fast-components/src/accordion/accordion.ts

```diff
@@ -0,0 +1,137 @@
+import { attr, FASTElement, observable } from "@microsoft/fast-element";
+import {
+    keyCodeArrowDown,
+    keyCodeArrowUp,
+    keyCodeEnd,
+    keyCodeHome,
+    wrapInBounds,
+} from "@microsoft/fast-web-utilities";
+import { StartEnd } from "../patterns/start-end";
+import { applyMixins } from "../utilities";
+import { FASTAccordionItem } from "./accordion-item";
+
+export enum AccordionExpandMode {
+    single = "single",
+    multi = "multi",
+}
+
+export class Accordion extends FASTElement {
+    @attr
+    public expandmode: AccordionExpandMode = AccordionExpandMode.multi;
+
+    @observable
+    public accordionItems: HTMLElement[];
+    public accordionItemsChanged(oldValue, newValue): void {
+        if (this.$fastController.isConnected) {
+            this.removeHeaderListeners(oldValue);
+            this.accordionIds = this.getHeaderIds();
+            this.setItems();
+        }
+    }
+
+    private activeid: string;
+    private activeHeaderIndex: number = 0;
+    private accordionIds: Array<string | null>;
+
+    private change = (): void => {
+        this.$emit("change", this.activeid);
```

Comment:
same comment on the above here - this data is accessible from the event target so maybe it's unnecessary?